### DISCUSSION TITLE
How to hide axis labels/text

### DISCUSSION THREAD

**Discussion by Symbai** [oxyplot/oxyplot#1947] [Q&A]
`IsAxisVisible` will hide the labels/text of the axis but also the grid lines. I want to show grid lines but no ticks and no labels. Ticks I can disable by setting `TickStyle="None"` but how can I disable the labels/text?

**Comment by VisualMelon** [comment]
I don't think there is a trivial way to disable label rendering'; the title can be removed by setting it to `null`. You can override the class and override `GetTickValues` so that it doesn't return any labels, but that's no fun.

**Reply by Symbai** [reply]
as a workaround I am using label formatter and return empty string. However oxyplot still waste resources by creating the labels and doing measuring, calling the formatter etc. We can disable many things but not labels. There should be an easy way
